charts/internal/shoot-system-components/charts/rook-ceph/templates/ceph-blockpool.yaml

```diff
@@ -0,0 +1,76 @@
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+  name: replicapool
+  namespace: rook-ceph
+spec:
+  failureDomain: host
+  replicated:
+    size: 3
+    # Disallow setting pool with replica 1, this could lead to data loss without recovery.
+    # Make sure you're *ABSOLUTELY CERTAIN* that is what you want
+    requireSafeReplicaSize: true
+    # gives a hint (%) to Ceph in terms of expected consumption of the total cluster capacity of a given pool
+    # for more info: https://docs.ceph.com/docs/master/rados/operations/placement-groups/#specifying-expected-pool-size
+    #targetSizeRatio: .5
+---
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
```

Comment:
The `StorageClass`es should be put here: https://github.com/gardener/gardener-extension-provider-packet/blob/master/charts/internal/shoot-storageclasses/templates/storageclasses.yaml

Reply:
Moved, but not resolving the conversation until you have a chance to review it as well.